### PR TITLE
Issue1471 Chiller staging subsequences: Capacities

### LINKED ISSUE #1471: Chiller staging: capacities
Implement the following chiller staging sequences:

- [x] Capacity requirement.
- [x] Design and minimal capacities.

The outputs of these sequences are inputs into the calculation of operating and staging part load ratios.

### PR DESCRIPTION
**Pull request by milicag**
This should be reviewed after [PR#1454](https://github.com/lbl-srg/modelica-buildings/pull/1454) is merged.

Closes #1471.

### CODE REVIEW

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/CapacityRequirement.mo [hunk 1]
```diff
@@ -0,0 +1,177 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Subsequences;
+block CapacityRequirement
+  "Cooling capacity requirement"
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/CapacityRequirement.mo]: In section 5.2.4.7, it states that "...a 5-minutes rooling average of instantaneous values sampled at a minimum of every 30 seconds." 
Should this sampling be implemented in this sequence?

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Configurator.mo [hunk 2]
```diff
@@ -144,52 +144,52 @@ protected
...
+  connect(staType.y, pro.u1) annotation (Line(points={{-178,-60},{-160,-60},{-160,
+          -74},{-142,-74}},color={0,0,127}));
+  connect(staDesCaps.y, yDesCap) annotation (Line(points={{-118,150},{100,150},{
+          100,60},{240,60}}, color={0,0,127}));
+  connect(staMinCaps.y, yMinCap) annotation (Line(points={{-118,110},{80,110},{80,
+          20},{240,20}}, color={0,0,127}));
   connect(oneVec.y, sumNumChi.u)
-    annotation (Line(points={{-179,60},{-142,60}}, color={0,0,127}));
+    annotation (Line(points={{-178,60},{-142,60}}, color={0,0,127}));
   annotation (defaultComponentName = "conf",
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Configurator.mo]: in the info section, the section number are different between difference version of RP1711. For later version (March and July versions):
- section 5.2.4.9 should be 5.2.4.12
- section 5.2.4.10 should be 5.2.4.13
- there is no design stage capacity anymore in July version, section 3.1.1.4, 2

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Status.mo [hunk 3]
```diff
@@ -226,127 +226,127 @@ protected
...
+  connect(reaToInt.y, intSwi.u3) annotation (Line(points={{82,110},{90,110},{90,
+          90},{160,90},{160,102},{178,102}}, color={255,127,0}));
+  connect(reaToInt1.y, intSwi1.u3) annotation (Line(points={{82,-70},{90,-70},{90,
+          -100},{170,-100},{170,-78},{178,-78}},    color={255,127,0}));
   connect(intSwi3.y, yUp)
-    annotation (Line(points={{381,80},{450,80}}, color={255,127,0}));
+    annotation (Line(points={{382,80},{460,80}}, color={255,127,0}));
   annotation (defaultComponentName = "sta",
         Icon(graphics={
         Rectangle(
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Status.mo]: Section number has been changed from 5.2.4.11.3 to 5.2.4.14.3.

**Comment by JayHuLBL** [comment]
@milicag I have added all commments I have now. It's ready to merge after addressing the comments and CI tests passed.

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/CapacityRequirement.mo (hunk 1 above)
Comment (milicag) [reply Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/CapacityRequirement.mo]: yes, it's already in there, as a `movMea` block.

Comment (milicag) [reply Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/CapacityRequirement.mo]: I see. I think the inputs need to be values that are sampled at at least the 30s rate. If we would sample them anywhere inside of this sequence, that would not guarantee that the quoted statement is satisfied.

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Configurator.mo (hunk 2 above)
Comment (milicag) [reply Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Configurator.mo]: Inroduced the first two points. For the third, they still in the July version use "design capacity of Chiller X", so I'll use "design stage capacity", which is the sum of all design capacities of chillers in that stage.

Comment (milicag) [reply Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Configurator.mo]: I double-checked the "stage design capacity" construct, and confirmed with version July, 5.2.4.5. wording.

**Pull request merged**